### DISCUSSION TITLE
My-CS-Notebook/Gossip/2023%E5%B9%B4%E5%BA%A6%E6%80%BB%E7%BB%93/

### DISCUSSION THREAD

**Discussion by giscus** [Zhang-Each/My-CS-Notebook#7] [General]
# My-CS-Notebook/Gossip/2023%E5%B9%B4%E5%BA%A6%E6%80%BB%E7%BB%93/

https://zhang-each.github.io/My-CS-Notebook/Gossip/2023%E5%B9%B4%E5%BA%A6%E6%80%BB%E7%BB%93/

**Comment by sgzerolc** [comment]
大概清醒的人最烦恼
